scripts/generator/templates/template-vaadin-platform-javadoc-pom.xml

```diff
@@ -25,6 +25,13 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>jakarta.platform</groupId>
+                <artifactId>jakarta.jakartaee-bom</artifactId>
+                <version>${jakarta.ee.version}</version>
```

Comment:
where this `jakarta.ee.version` come after removing it from vaadin-bom?

Reply:
Platform parent POM

Reply:
ah. my bad... it is from parent pom...